### DISCUSSION TITLE
Errors importing CISCO_IOS_SWITCH_L2S_STIG v2R4

### DISCUSSION THREAD

**Discussion by mcklutty35** [NUWCDIVNPT/stig-manager#1210] [General]
When importing the fully assessed cklb for this STIG into StigManager it shows up with no results applied at all (0% assessment). Exporting (from StigViewer) to a ckl and loading that into Stigmanager gives us an error on 3 of the rules (e.g. No grant for this asset/ruleid  SV-220581r879565) but the rest look correct. However, loading either the cklb or clk into StigViewer (v3) shows all rules having been assessed correctly as expected.  What would cause StigManager to behave this way?

**Comment by cd-rite** [comment]
Hi @mcklutty35 It sounds like you might not have that version of the STIG installed in STIG Manager.

**Comment by mcklutty35** [comment]
Hmm, I'll have to verify the version of the benchmark that was run vs what's in StigManager.  But why would the exact same checklist in .cklb format show 0% assessed vs that same checklist in .ckl format showing 93% assessed?

**Reply by cd-rite** [reply]
Ah, ok. That is likely an issue with STIG Viewer 3.... See my response to issue #1206 here:
https://github.com/NUWCDIVNPT/stig-manager/issues/1206#issuecomment-1921971591

TLDR: In some cases, STIG Viewer will generate .cklb files with only the rule_id property specified and not rule_id_src. Their schema says "rule_id" is for display, and "rule_id_src" is the actual rule_id, so that's the field we look for to import reviews.

**Reply by cd-rite** [reply]
Hi @mcklutty35 We were able to add some processing to account for the STIG Viewer .cklb anomalies you found.  Our latest release, 1.4.3, should handle them fine, now.
